app/_includes/nav-v2.html

```diff
@@ -2,7 +2,9 @@
   <a class="skip-main" href="#main">Skip to content</a>
   <div id="promo-banner">
     <div class="container">
-      <div class="closebanner"></div> <strong><label>New</label> Konnect Plus: Pay-as-you-go from startup friendly to enterprise scale. <a target="_blank" href="https://konghq.com/pricing/">See Pricing &rarr;</a></strong></div>
+      <div class="closebanner"></div> <strong><label>EVENT</label> Kong Summit agenda announced! <a target="_blank"
+          href="https://konghq.com/kong-summit/#register">See Pricing &rarr;</a></strong>
```

Comment:
This link + link label don't really match. It says "See pricing", but the link takes you to a registration form with no mention of cost. Might it be better to either link to the top of the page (https://konghq.com/kong-summit), where you can see that it says "Free", or change the label to remove any mention of pricing?